wsi_deid/web_client/views/ItemView.js

```diff
@@ -102,15 +109,61 @@ wrap(ItemView, 'render', function (render) {
         return isSquare && $(event.target).is('input[type="checkbox"]');
     };
 
-    const showRedactButton = (keyname) => {
-        if (keyname.match(/^internal;aperio_version$/)) {
-            return false;
+    const getFormat = () => {
+        if (this.$el.find('.large_image_metadata_value[keyname^="internal;openslide;aperio."]').length > 0) {
```

Comment:
In the Python, we test if the key `openside.vendor` is one of `aperio` or `hamamatsu`.  If think here that test would be `this.$el.find('.large_image_metadata_value[keyname="internal;openslide;openslide.vendor"]').text() === 'aperio'`. We should make these the same.